pkg/agent/openflow/framework.go

```diff
@@ -244,10 +252,16 @@ func (f *featureEgress) getRequiredTables() []*Table {
 }
 
 func (f *featureMulticast) getRequiredTables() []*Table {
-	return []*Table{
+	tables := []*Table{
 		MulticastRoutingTable,
 		MulticastOutputTable,
 	}
+	if f.enableAntreaPolicy {
+		tables = append(tables,
+			MulticastIGMPIngressTable,
```

Comment:
why adding this table alone? I assume this is kind of duplicate with required tables of feature networkpolicy, but don't understand why it's just this table. If these tables are only used by feature NetworkPolicy's methods, I think we just add to that feature